src/algorithms/ctmrg/ctmrg.jl

```diff
@@ -92,6 +92,31 @@ function _singular_value_distance((S₁, S₂))
     end
 end
 
+function build_projectors(
+    U::AbstractTensorMap{E,2,1},
+    S::AbstractTensorMap{E,1,1},
+    V::AbstractTensorMap{E,1,2},
+    Q::AbstractTensorMap{E,2,2},
+    Q_next::AbstractTensorMap{E,2,2},
+) where {E<:ElementarySpace}
+    isqS = sdiag_pow(S, -0.5)
+    P_left = Q_next * V' * isqS
+    P_right = isqS * U' * Q
+    return P_left, P_right
+end
```

Comment:
This looks like it should be the same implementation for PEPS?

Reply:
Missed the fact that I should have removed this one when rebasing onto the current master I think, `build_projectors` must have changed name in the meantime and the implementation is indeed network-agnostic.